src/_guides/language/effective-dart/documentation.md

```diff
@@ -251,14 +251,14 @@ void start() {
 }
 {% endprettify %}
 
-### PREFER starting variable, getter, or setter comments with noun phrases.
+### PREFER starting non-boolean variables or properties with noun phrases.
```

Comment:
I think it's important to keep "comment" in here, because it's the comment this applies to and not the member itself. How about:

```
### PREFER starting a non-boolean variable or property comment with a noun phrase.
```

Reply:
Woops, didn't mean to remove it! Fixed.